lib/process-services-cloud/src/lib/task/task-filters/components/edit-task-filters/base-edit-task-filter-cloud.component.html

```diff
@@ -0,0 +1,108 @@
+<mat-accordion [hideToggle]="isLoading">
+    <mat-expansion-panel (afterExpand)="onExpand()"
+                         (closed)="onClose()">
+        <mat-expansion-panel-header *ngIf="taskFilter"
+                                    id="adf-edit-task-filter-expansion-header">
+            <ng-container *ngIf="!isLoading; else loadingTemplate">
+                <mat-panel-title *ngIf="showTaskFilterName"
+                                 fxLayoutAlign="space-between center"
+                                 id="adf-edit-task-filter-title-id">{{taskFilter.name | translate}}</mat-panel-title>
+                <mat-panel-description fxLayoutAlign="space-between center"
+                                       id="adf-edit-task-filter-sub-title-id">
+                    <span *ngIf="showTitle">{{ 'ADF_CLOUD_EDIT_TASK_FILTER.TITLE' | translate}}</span>
+                    <div *ngIf="showActions()"
+                         class="adf-cloud-edit-task-filter-actions">
+                        <ng-container *ngIf="toggleFilterActions">
+                            <button *ngFor="let filterAction of taskFilterActions"
+                                    mat-icon-button
+                                    matTooltip="{{ filterAction.tooltip | translate}}"
+                                    [attr.data-automation-id]="'adf-filter-action-' + filterAction.actionType"
+                                    [disabled]="isDisabledAction(filterAction)"
+                                    (click)="executeFilterActions(filterAction)">
+                                <mat-icon>{{filterAction.icon}}</mat-icon>
+                            </button>
+                        </ng-container>
+                    </div>
+                </mat-panel-description>
+            </ng-container>
+            <ng-template #loadingTemplate>
+                <div class="adf-cloud-edit-task-filter-loading-margin">
+                    <mat-spinner [diameter]="30"></mat-spinner>
+                </div>
+            </ng-template>
+        </mat-expansion-panel-header>
+        <ng-container *ngIf="!isLoading;">
+            <form [formGroup]="editTaskFilterForm"
+                  *ngIf="editTaskFilterForm">
+                <div fxLayout="row wrap"
```

Comment:
can you try using vanilla css instead of flex-layout? this library is going to be deprecated in the future and I spent really lot of time cleaning up this stuff already